transport/rif_comms/client.py

```diff
@@ -102,16 +93,3 @@ def disconnect(self):
         """
         self.stub.EndCommunication(Void())
         self.grpc_channel.unsubscribe(lambda: self.grpc_channel.close())
-
-    def _get_peer_id(self, rsk_address: Address) -> str:
```

Comment:
im ok with deleting this since we are not using anymore, but, i also think we can use this call (using our address as rsk_address) as a kind of healhtcheck or ping (to know when rif comms is ready)

probably a ping or healtcheck special endpoint would be nicer, but we can consider this option if needed

Reply:
sure. let's add it back when we need it, otherwise unused functions just bloat up the codebase even more